### ISSUE TITLE
About package name

### ISSUE THREAD

**Issue by Ghost-chu** [Bilicraft-Community/StargateDynmap#8]
Hi, everyone in the Stargate community!  
It's great to see that this project is still being actively updated! I've noticed that this repository still seems to be using the `com.ghostchu` package, however I haven't continued to maintain it for a long time after I handed over the organization to Stargate. So it's possible that a large portion of the code has been replaced with better versions.

While I personally don't mind at all if I continue to use the com.ghostchu package name, I'm more than happy to see this repository move away from my personal figure and into the hands of a better community.

If this is due to a lack of authorization to do so, I'd be happy to provide written authorization (including change of package name, author name, community name, and open source license agreement).

In the meantime, this project is no longer part of Bilicraft and this section can be removed, it is now free.

In any case, looking forward to hearing any replies :)

**Comment by Ghost-chu** [comment]
TL;DR: It's good to see that it's still being actively developed, so go ahead and change the `com.ghostchu` package name to make it fully community owned. ;)

**Comment by Thorinwasher** [comment]
Sure I will change the package name

**Comment by Ghost-chu** [comment]
> Sure I will change the package name

Glad to see the community is still active, great job! Cheers!
